### PR TITLE
improvement(grow_shrink_cluster): few small improvements

### PR DESCRIPTION
**Pull request by juliayakovlev**
1. if nemesis failed, print failure message. Now we always get successful message
2. Add Disruption event when Grow cluster part is finished

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -2670,11 +2670,8 @@ def decommission_nodes(self, add_nodes_number, rack, is_seed: Optional[Union[boo
             else:
                 self.set_target_node(is_seed=is_seed)
             self.log.info("Next node will be removed %s", self.target_node)
-            try:
-                InfoEvent(message='StartEvent - ShrinkCluster started decommissioning a node').publish()
```
Comment (roydahan) [inline sdcm/nemesis.py:2674]: This is not good, we need this for the latency calculations in performance tests.
+ it's much clearer than the nodetool event.

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -2670,11 +2670,8 @@ def decommission_nodes(self, add_nodes_number, rack, is_seed: Optional[Union[boo
...
-                self.decommission_node(self.target_node)
-            finally:
-                InfoEvent(message='FinishEvent - ShrinkCluster has done decommissioning a node').publish()
```
Comment (roydahan) [inline sdcm/nemesis.py:2677]: same as above.

File: test-cases/longevity/longevity-10gb-3h.yaml [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 
-instance_type_db: 'i3.4xlarge'
+instance_type_db: 'i3.2xlarge'
 gce_instance_type_db: 'n1-highmem-16'
 gce_instance_type_loader: 'e2-standard-4'
 scylla_repo_loader: 'https://s3.amazonaws.com/downloads.scylladb.com/rpm/centos/scylla-4.3.repo'
 
-nemesis_class_name: 'SisyphusMonkey'
+nemesis_class_name: 'GrowShrinkClusterNemesis'
 nemesis_seed: '111'
```
Comment (roydahan) [inline test-cases/longevity/longevity-10gb-3h.yaml:15]: This shouldn't be here!
This is a dangerous mistake!

**Review by roydahan — CHANGES_REQUESTED** [review]
See my comments.

File: test-cases/longevity/longevity-10gb-3h.yaml (hunk 3 above)
Comment (juliayakovlev) [reply test-cases/longevity/longevity-10gb-3h.yaml:15]: It's not mistake. This PR is not ready for review

File: sdcm/nemesis.py (hunk 1 above)
Comment (juliayakovlev) [reply sdcm/nemesis.py:2674]: fixed

File: sdcm/nemesis.py (hunk 2 above)
Comment (juliayakovlev) [reply sdcm/nemesis.py:2677]: fixed

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by bentsi — APPROVED** [review]

**Pull request merged**